cmd/zc_enumerator.go

```diff
@@ -46,7 +46,7 @@ import (
 // represent a local or remote resource object (ex: local file, blob, etc.)
 // we can add more properties if needed, as this is easily extensible
 // ** DO NOT instantiate directly, always use newStoredObject ** (to make sure its fully populated and any preprocessor method runs)
-type storedObject struct {
+type StoredObject struct {
 	name             string
```

Comment:
Were these properties missed?

Reply:
No Ze, Till now sample code not using any of its properties. May be as i add more functionality i need those properties. Then i will export them. Or i can write function in cmd which work on these properties.